.github/workflows/_build_artifacts.yml

```diff
@@ -236,7 +236,7 @@ jobs:
         uses: docker/metadata-action@v5
         with:
           images:
-            ${{ steps.login.outputs.registry }}/firezone/${{ matrix.name.image_name }}${{ inputs.image_suffix }}
+            ${{ steps.login.outputs.registry }}/firezone/${{ inputs.image_prefix && format('{0}/', inputs.image_prefix) }}${{ matrix.name.image_name }}
```

Comment:
What does this syntax do? `inputs.image_prefix && format('{0}/', inputs.image_prefix)`

Reply:
Adds `/` to `image_prefix` if `image_prefix` was defined, otherwise outputs `""`